README.md

```diff
@@ -1,387 +1,58 @@
-supports syntax highlighting, auto-completion and many other features.*
-
-## Creating keybindings for R commands
-
-There are 3 ways you can use extension functions to create keybindings that run R commands in the terminal:
-
-1. `r.runCommand` to make a keybinding to run any R expression.
-2. `r.runCommandWithEditorPath` to create a keybinding for an R expression where the placeholder value `$$` is interpolated with the current file path.
-3. `runCommandWithSelectionOrWord` to create a keybinding for an R expression where `$$` is interpolated with the current selection or the current word the cursor is on.
-
-Here are some example entries from `keybindings.json`:
-
-```json
-[
-    {
-        "description": "run drake::r_make()",
-        "key": "ctrl+;",
-        "command": "r.runCommand",
-        "when": "editorTextFocus",
-        "args": "drake::r_make()"
-    },
-    {
-        "description": "load drake target at cursor",
-        "key": "ctrl+shift+;",
-        "command": "r.runCommandWithSelectionOrWord",
-        "when": "editorTextFocus",
-        "args": "drake::loadd($$)"
-    },
-    {
-        "description": "knit to html",
-        "key": "ctrl+i",
-        "command": "r.runCommandWithEditorPath",
-        "when": "editorTextFocus",
-        "args": "rmarkdown::render(\"$$\", output_format = rmarkdown::html_document(), output_dir = \".\", clean = TRUE)"
-    }
-]
-```
-
-## TODO
-
-* Debug
-
-## CONTRIBUTING
-
-* Please see [CONTRIBUTING.md](https://github.com/Ikuyadeu/vscode-R/blob/master/CONTRIBUTING.md)
-
-This extension based on
-
-* [r.tmbundle](https://github.com/textmate/r.tmbundle)
-* [markdown-redcarpet.tmbundle](https://github.com/streeter/markdown-redcarpet.tmbundle)
-* [Markdown extension in VS Code](https://github.com/Microsoft/vscode/blob/master/extensions/markdown/snippets/markdown.json)
-* [R.gitignore](https://github.com/github/gitignore/raw/master/R.gitignore)
-* [language-r](https://github.com/lee-dohm/language-r)
-* [R box](https://github.com/randy3k/R-Box)
-
-## Collaborators
-
-I hope you will join us.
-
-* [@andycraig](https://github.com/andycraig)
-* [@Ladvien](https://github.com/Ladvien)
-* [@renkun-ken](https://github.com/renkun-ken)
-* [@MilesMcBain](https://github.com/MilesMcBain)
-* [@ManuelHentschel](https://github.com/ManuelHentschel)
-* [@ElianHugh](https://github.com/ElianHugh)
-
-## FAQ
-
-* Q: I can't use command and message is `xxx no command found`.
-* A: Please open your folder that has R source file
-
-* Q: About code formatter, completion, definition...
-* A: Please visit to the language server [issues](https://github.com/REditorSupport/languageserver/issues)
-
-Other past questions can be found from [StackOverflow](https://stackoverflow.com/questions/tagged/visual-studio-code+r) or [issues](https://github.com/Ikuyadeu/vscode-R/issues)
+## Features
 
-The R logo is © 2016 The R Foundation
+* Extended syntax highlighting for R, R Markdown and R Documentation.
```

Comment:
Could you make the space like a python extension for each item?
All of them are many great contributions but look a bit complicated.